roles/identity-management/manage-rh-sso-identities/tasks/main.yml

```diff
@@ -8,4 +8,16 @@
   when:
     - identities.targets is undefined or 'rh-sso' in identities.targets
   tags:
-    - always
+    - create-user
```

Comment:
Do we need create-* tags? Shouldn't the default behavior just be create and then we should delete only when requested (via the tag)